ipatests/test_integration/test_installation.py

```diff
@@ -196,3 +197,36 @@ def test_install_master(self):
 
     def test_install_kra(self):
         tasks.install_kra(self.master, first_instance=True)
+
+
+@pytest.mark.ds_acceptance
```

Comment:
This looks like just simple_replication test, why don't you mark it instead?

Simple replication test even tests if replication is working properly, not just installation.

I would to extend simple_replication to test both cases replication of domain, and replication of ca.